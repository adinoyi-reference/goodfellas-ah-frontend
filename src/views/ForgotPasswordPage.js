
import React from 'react';
// import NewPassword from '../components/layout/NewPassword';
import Forgotpassword from '../containers/Forgotpassword';
import Reset from '../components/layout/Reset';

const ForgotPasswordPage = (props) => (
  <div>
    <Reset>
      <Forgotpassword {...props} />
    </Reset>
  </div>
);

export default ForgotPasswordPage;
