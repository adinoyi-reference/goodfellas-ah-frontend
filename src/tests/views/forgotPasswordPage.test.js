import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import ForgotPasswordPage from '../../views/ForgotPasswordPage';

let wrapped;

beforeEach(() => {
  wrapped = shallow(<ForgotPasswordPage />);
});

describe('ForgotPasswordPageUI', () => {
  describe('render features', () => {
    test('view should render as expected', () => {
      const tree = toJson(wrapped);
      expect(tree).toMatchSnapshot();
    });
  });
});
