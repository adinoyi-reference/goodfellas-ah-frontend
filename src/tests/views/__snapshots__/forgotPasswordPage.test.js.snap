// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ForgotPasswordPageUI render features view should render as expected 1`] = `
<div>
  <Auth>
    <Connect(ForgotPassword) />
  </Auth>
</div>
`;
